crates/cast/src/cmd/logs.rs

```diff
@@ -269,7 +269,7 @@ mod tests {
 
     #[test]
     fn test_build_filter_sig_with_arguments() {
-        let addr = Address::from_str(ADDRESS).unwrap();
+        let addr = address!(ADDRESS);
```

Comment:
could we update the function def instead? like instead of defining this as str we can just ADDRESS: Address = address!